### PR TITLE
Include project fluent-plugin-openlineage

### PR DESCRIPTION
**Pull request by jonathanlbt1**
This branch contains the folder with the first release of fluent-plugin-openlineage, accessible at RubyGems ( https://rubygems.org/gems/fluent-plugin-openlineage ). The original material in this folder was created by Pawel Leszczynski, to whom I am very grateful. All modifications to his work were made exclusively to develop this atomic plugin, which can now be easily used with a Dockerfile and a configuration file.

### CODE REVIEW

**Comment by mobuchowski** [comment]
@jonathanlbt1 why is it different than https://github.com/OpenLineage/OpenLineage/tree/main/proxy/fluentd? Shouldn't the changes be made to that directory rather than adding separate one?

File: integration/spark/docker-compose.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+  #   image: jupyter/pyspark-notebook:spark-3.1.2
+  #   ports:
+  #     - "8888:8888"
+  #   volumes:
+  #     - ./docker/notebooks:/home/jovyan/notebooks
+  #     - ./build:/home/jovyan/openlineage
+  #   links:
+  #     - "api:marquez"
+  #   depends_on:
+  #     - api
```
Comment (mobuchowski) [inline integration/spark/docker-compose.yml:14]: Not sure why it's commented out in this PR.

**Comment by jonathanlbt1** [comment]
> @jonathanlbt1 why is it different than https://github.com/OpenLineage/OpenLineage/tree/main/proxy/fluentd? Shouldn't the changes be made to that directory rather than adding separate one?

Hi! I just wanted you to have a look at the changes first. I'm preparing some changes to a future PR. This one is only for you to look at the changes and give me suggestions. It's not meant to be accepted into the project yet. Since I didn't know how to share that, I opened a PR into another folder to preserve the work already done in the community.

File: integration/spark/docker-compose.yml (hunk 1 above)
Comment (jonathanlbt1) [reply integration/spark/docker-compose.yml:14]: Please, just ignore it. It's not meant to be accepted yet. :-)

**Comment by pawel-big-lebowski** [comment]
@jonathanlbt1 Is it then a draft PR at the moment? If so, please convert it to draft.

**Pull request closed without merging**